test/integration/test-helper.coffee

```diff
@@ -0,0 +1,39 @@
+_ = require 'underscore'
+Promise = require 'bluebird'
+
+module.exports.unpublishProduct = (logger, client, product) ->
```

Comment:
Actually a lot nicer imho is declaring the functions and export all of them in one place:
```js
module.exports = {
  unpublishProduct,
  (...)
}
```

And using `module.exports` instead of `exports` is better just for the sake of consistency. (You can't use `exports` to export just a function.)